### ISSUE TITLE
1.7.10 version crashes on a server

### ISSUE THREAD

**Issue by brandyyn** [ThexXTURBOXx/balkons-weaponmod#34]
works in SP but crashes when trying to start a server with it 

[crash-2024-06-26_19.09.57-server.txt](https://github.com/user-attachments/files/15993591/crash-2024-06-26_19.09.57-server.txt)

**Comment by ThexXTURBOXx** [comment]
Thanks for the report! Looks like the next release is gonna focus on server crash fixes :)
Will fix this tomorrow as I am on a business trip currently

**Comment by brandyyn** [comment]
> Thanks for the report! Looks like the next release is gonna focus on server crash fixes :) Will fix this tomorrow as I am on a business trip currently

no rush! i'm using the original version in the mean time. do you play to add anything to the 1.7 version?

**Comment by ThexXTURBOXx** [comment]
Thanks, and enjoy! :)
I am planning on extending the collection of weapons, but first I want to fix all the bugs that can be found - there are probably quite a few, some of which may even be race conditions or interactions with other mods, new entities or items from modern Minecraft versions etc.
Let's see how everything goes!

**Comment by ThexXTURBOXx** [comment]
By the way, at first I was a bit puzzled why the game crashes with this weird error, but then I remembered what I did:
I implemented a hacky way to load the unused sound `random.breath` into the game. For that, I did some stuff with the `resourceManager`, which is only available on the client!
One of the stupidest mistakes I ever made, honestly... I even call `Minecraft.getMinecraft()` on the server! At least, it's a quick fix :)
